Gemfile

```diff
@@ -3,6 +3,11 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in curation_concerns.gemspec
 gemspec
 
+gem 'hydra-works', github: 'projecthydra/hydra-works', branch: 'member_of'
+gem 'active-fedora', github:'projecthydra/active_fedora', branch: 'master'
```

Comment:
I think `active-fedora` and `hydra-derivatives` can point at released versions now.